src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -221,11 +233,354 @@ function svdTTN(
   return H
 end
 
+function qn_svdTTN(
+  os::OpSum{C}, sites::IndsNetwork{VT,<:Index}, root_vertex::VT; kwargs...
+)::TTN where {C,VT}
+  # Function barrier to improve type stability
+  ValType = ITensors.determineValType(terms(os))
+  return qn_svdTTN(ValType, os, sites, root_vertex; kwargs...)
+end
+
+function qn_svdTTN(
+  ValType::Type{<:Number},
+  os::OpSum{C},
```

Comment:
```suggestion
  os::OpSum,
```

Reply:
Yeah it is a bit unfortunate, but the reason for that is because `OpSum()` defaults to complex coefficients so that users can add terms in-place if the coefficients are real or complex (users could force it to be real with `OpSum{Float64}()` but then of course can't add complex coefficients in-place).

So yeah, the `coefficient_type` input is meant to help fix that by determining the most narrow coefficient type at runtime and then using those kinds of conversions to narrow the coefficients if needed.

Reply:
I see that `C` is used in a few places in this function, hopefully we can just use `coefficient_type` instead.

Reply:
Yes, that works, I already fixed this locally.